csharp-intermediate-topics/ConstVsReadonlyInCSharp/Tests/MathCalculatorUnitTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using ConstVsReadonlyInCSharp;
+
+namespace Tests
+{
+    public class MathCalculatorUnitTests
+    {
+        private MathCalculator mathCalculator = new MathCalculator();
+
+        [Fact]
+        public void WhenAdding1And3_TheReturn4()
+        {
+            var result = mathCalculator.Add(1, 3);
+
+            Assert.Equal((decimal)4.0, result);
```

Comment:
See previous comments about floating point precision and equality, although decimal is precise to 28 decimal places, it still performs rounding and so can still produce incorrect equality results. Also, no need for the cast, this can be written as a decimal literal with M